### PR TITLE
fix for computing jacobian when the root_joint is not an active joint

### PR DESCRIPTION
**Pull request by sachinchitta**

### CODE REVIEW

File: robot_state/src/robot_state.cpp [hunk 1]
```diff
@@ -975,7 +975,7 @@ bool moveit::core::RobotState::getJacobian(const JointModelGroup *group, const L
     return false;
   }
 
-  const robot_model::JointModel* root_joint_model = group->getJointRoots()[0];
+  const robot_model::JointModel* root_joint_model = group->getJointModels()[0];//group->getJointRoots()[0];
```
Comment (isucan) [inline robot_state/src/robot_state.cpp:978]: I am assuming you are doing this to account for the case when a root joint is fixed, correct?

Comment (hersh) [inline robot_state/src/robot_state.cpp:978]: yes, he is.

Comment (hersh) [inline robot_state/src/robot_state.cpp:978]: (he is here now)

**Comment by sachinchitta** [comment]
This should also work for the case when the root joint is a mimic joint.

File: robot_state/src/robot_state.cpp (hunk 1 above)
Comment (isucan) [inline robot_state/src/robot_state.cpp:978]: sounds good. Just checking that was indeed the intention :)

**Pull request merged**